letsencrypt/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 5.3.4
+
+- Improve documentation
```

Comment:
The documentation is part of the store metadata, which is fetched regularly by Supervisor. This means there is no actual version bump needed.

Reply:
Thanks for the hint.